mongo/mongo.php

```diff
@@ -375,21 +375,21 @@ public function pairPersistConnect($username = "", $password = "") {}
      */
     protected function connectUtil() {}
 
-   /**
-    * Check if there was an error on the most recent db operation performed
-    * @link https://secure.php.net/manual/en/mongo.lasterror.php
-    * @return array|null Returns the error, if there was one, or NULL.
-    * @see MongoDB::lastError()
-    */
+    /**
+     * Check if there was an error on the most recent db operation performed
+     * @link https://secure.php.net/manual/en/mongo.lasterror.php
+     * @return array|null Returns the error, if there was one, or NULL.
+     * @see MongoDB::lastError()
+     */
    #[Deprecated('Use MongoDB::lastError() instead.')]
     public function lastError() {}
 
-   /**
-    * Checks for the last error thrown during a database operation
-    * @link https://secure.php.net/manual/en/mongo.preverror.php
-    * @return array Returns the error and the number of operations ago it occurred.
-    * @see MongoDB::prevError()
-    */
+    /**
```

Comment:
seems to be automatic cs fixes

Reply:
Hmm.. I'm not sure about that. Probably it was reformated by IDE according to your code style settings before commit. Please check if you have the option "Reformat Code" enabled in the commit dialog and disable it if so, then call `composer cs-fix` just in case and push after that

Reply:
Well, on my end I did not run any IDE auto fix change and ran `composer cs-fix` after all my changes, but they want to stick to this state
What PHP version are you running ?